scripts/core_build.sh

```diff
@@ -55,8 +55,9 @@ GPU_VERSION="OFF" #defaults to CPU version
 WITH_PROMETHEUS="ON"
 CUDA_ARCH="DEFAULT"
 CUSTOM_THIRDPARTY_PATH=""
+EMBEDDED_MILVUS="OFF"
 
-while getopts "p:d:t:s:f:ulrcghzme" arg; do
+while getopts "b:p:d:t:s:f:ulrcghzme" arg; do
```

Comment:
If it's an "OFF/ON" option, this should be changed into `b` (not `b:`), so that `./buils.sh -b` would turn the  EMBEDED_MILVUS=ON. Please refer to `GPU_VERSION` as an example.

In your implementation, we'll have to use `./buils.sh -b ON` to switch it ON.

Reply:
Done.